### PR TITLE
Create MailPiece.dart

### PR DESCRIPTION
**Pull request by sHuber125**
sample commit, test

### CODE REVIEW

File: lib/models/MailPiece.dart [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class MailPiece {
+  int _mail_ID;
+  DateTime _informedDate;
+  String _LogoText;
+  AddressObject _senderAddress;
```
Comment (umgcjack) [inline lib/models/MailPiece.dart]: These all need to be public (not prefixed with `_`).
`logoText` and `mailId` should be lowerCamelCased.

File: lib/models/MailPiece.dart [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+   MailPiece( this._mail_ID, this._informedDate, this._LogoText, this._senderAddress );
+
+
```
Comment (umgcjack) [inline lib/models/MailPiece.dart]: You could probably remove these, and it wouldn't hurt to end the file in a newline.

**Review by umgcjack — CHANGES_REQUESTED** [review]

File: lib/models/MailPiece.dart (hunk 1 above)
Comment (umgcjack) [inline lib/models/MailPiece.dart]: I'd make this a `String`. Otherwise, you'll need to import whatever library provides the `AddressObject` type.

**Review by umgcjack — CHANGES_REQUESTED** [review]

**Comment by mdconatser** [comment]
It looks like a MailPiece class was already added (mistakenly to the root directory of the project). Recommend just moving that file into lib/models.

**Review by mdconatser — CHANGES_REQUESTED** [review]
It looks like a MailPiece class was already added (mistakenly to the root directory of the project). It looks like Shane's current PR moves this file to this location already, so recommend just abandoning this pr

**Comment by shaneknows** [comment]
PR is OBE

**Pull request closed without merging**